docker-compose.yml

```diff
@@ -164,11 +144,6 @@ services:
     ports:
       - "8080"
       - "8081"
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8081/readyz"]
-      interval: 30s
-      timeout: 10s
-      retries: 5
```

Comment:
These values do not match those used by Docker's HEALTHCHECK directive. Is that ok?

Reply:
see above